appengine/pom.xml

```diff
@@ -35,12 +35,11 @@
       <version>${appengine.target.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>javax.inject</groupId>
-      <artifactId>javax.inject</artifactId>
-      <version>1</version>
-      <scope>provided</scope>
-    </dependency>
+      <dependency>
```

Comment:
1) In my case it was enough to replace this dependency with `jakarta.inject:jakarta.inject-api:2.0.1` keepnig the same scope (provided)
2) It would be great to keep existing alignment for the dependency block (2 spaces for indention instead of 4)